src/styles/main.scss

```
@@ -0,0 +1,55 @@
+@use 'sass:list';
+@import './utils/variables';
+
+.page {
+  margin: 0;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: 7 * ($day-size + $gap) - $gap;
+  padding: 10px;
+
+  &__day {
+    box-sizing: border-box;
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid black;
+    background-color: #eee;
+    text-align: center;
+    line-height: 98px;
+    transition: $std-transition;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        font-family: $std-font-family;
+        font-size: $std-font-size;
+      }
+    }
```

```suggestion
    transition: $std-transition;
    
    &::before {
      font-family: $std-font-family;
      font-size: $std-font-size;
    }

    @for $i from 1 through 31 {
      &:nth-child(#{$i})::before {
        content: '#{$i}';
      }
    }
```
Consider adding these styles to a general selector instead of generating the same styling for each of the 31 elements